### PR TITLE
(DOCSP-42975) [Atlas CLI] Atlas CLI nested components

### PR DESCRIPTION
**Pull request by elyse-mdb**
### Unnesting 2 examples in an Atlas CLI page. 
[This spreadsheet](https://docs.google.com/spreadsheets/d/1M2IlsyV0-6mY-AvBBrX-4uSDtQrbt9zuoXsqIJaVqlk/edit?gid=0#gid=0) tracks the number of nested components on each page, and [this doc](https://docs.google.com/document/d/1yMbng-EPOQ82bAzBTC2VJwIvsUBOSCJh21cGbDoe1a8/edit#heading=h.7hxlffls9cx6)  provides guidelines for fixing nested components.

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [x] Resolve any new warnings or errors in the build.
- [x] Proofread for spelling and grammatical errors.
- [x] Check staging for rendering issues.
- [x] Confirm links are working.

### CODE REVIEW

**Comment by elyse-mdb** [comment]
Suggestion: Hold off merging and backporting the ticket until the ``important`` gold type setting is available.

File: source/atlas-cli-env-variables.txt [hunk 1]
```diff
@@ -119,46 +119,41 @@ The {+atlas-cli+} supports the following environment variables:
...
-             :copyable: false 
+         .. code-block:: sh 
+            :copyable: false 
 
-             HTTP_PROXY=<username>:<password>@<my.proxy.address>
+            HTTP_PROXY=<username>:<password>@<my.proxy.address>
 
-          The following example shows how to set up the environment 
-          variable if the scheme is ``socks5``.
+       - If the scheme is ``socks5``.
```
Comment (sarahsimpers) [inline source/atlas-cli-env-variables.txt:139]:
```suggestion
       - If the scheme is ``socks5``:
```

**Review by sarahsimpers — APPROVED** [review]
LGTM with one fix but the build failed, may need to raise with DOP or push an empty commit to retry

**Comment by elyse-mdb** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v1.29|[](https://github.com/mongodb/docs-atlas-cli/pull/686)|
|✅|v1.28|[](https://github.com/mongodb/docs-atlas-cli/pull/687)|
|✅|v1.27|[](https://github.com/mongodb/docs-atlas-cli/pull/688)|
|✅|v1.26|[](https://github.com/mongodb/docs-atlas-cli/pull/689)|
|✅|v1.25|[](https://github.com/mongodb/docs-atlas-cli/pull/690)|
|✅|v1.24|[](https://github.com/mongodb/docs-atlas-cli/pull/691)|
|✅|v1.23|[](https://github.com/mongodb/docs-atlas-cli/pull/692)|
|✅|v1.22|[](https://github.com/mongodb/docs-atlas-cli/pull/693)|
|✅|v1.21|[](https://github.com/mongodb/docs-atlas-cli/pull/694)|
|✅|v1.20|[](https://github.com/mongodb/docs-atlas-cli/pull/695)|
|✅|v1.19|[](https://github.com/mongodb/docs-atlas-cli/pull/696)|
|✅|v1.18|[](https://github.com/mongodb/docs-atlas-cli/pull/697)|
|✅|v1.17|[](https://github.com/mongodb/docs-atlas-cli/pull/698)|
|✅|v1.16|[](https://github.com/mongodb/docs-atlas-cli/pull/699)|
|✅|v1.15|[](https://github.com/mongodb/docs-atlas-cli/pull/700)|
|✅|v1.14|[](https://github.com/mongodb/docs-atlas-cli/pull/701)|
|✅|v1.13|[](https://github.com/mongodb/docs-atlas-cli/pull/702)|
|✅|v1.12|[](https://github.com/mongodb/docs-atlas-cli/pull/703)|
|✅|v1.11|[](https://github.com/mongodb/docs-atlas-cli/pull/704)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Comment by elyse-mdb** [comment]
Backport PRs: 

- v1.29: https://github.com/mongodb/docs-atlas-cli/pull/686
- v1.28: https://github.com/mongodb/docs-atlas-cli/pull/687
- v1.27: https://github.com/mongodb/docs-atlas-cli/pull/688
- v1.26: https://github.com/mongodb/docs-atlas-cli/pull/689
- v1.25: https://github.com/mongodb/docs-atlas-cli/pull/690
- v1.24: https://github.com/mongodb/docs-atlas-cli/pull/691
- v1.23: https://github.com/mongodb/docs-atlas-cli/pull/692
- v1.22: https://github.com/mongodb/docs-atlas-cli/pull/693
- v1.21: https://github.com/mongodb/docs-atlas-cli/pull/694
- v1.20: https://github.com/mongodb/docs-atlas-cli/pull/695
- v1.19: https://github.com/mongodb/docs-atlas-cli/pull/696
- v1.18: https://github.com/mongodb/docs-atlas-cli/pull/697
- v1.17: https://github.com/mongodb/docs-atlas-cli/pull/698
- v1.16: https://github.com/mongodb/docs-atlas-cli/pull/699
- v1.15: https://github.com/mongodb/docs-atlas-cli/pull/700
- v1.14: https://github.com/mongodb/docs-atlas-cli/pull/701
- v1.13: https://github.com/mongodb/docs-atlas-cli/pull/702
- v.1.12: https://github.com/mongodb/docs-atlas-cli/pull/703
- v1.11: https://github.com/mongodb/docs-atlas-cli/pull/704

**Pull request merged**